composer.json

```diff
@@ -1,12 +1,16 @@
 {
-    "name": "jeckel-lab/clock",
+    "name": "wenowfr/clock",
```

Comment:
`wenow` is more preferable for package name.

Reply:
True. Fixed.